functions.php

```diff
@@ -178,69 +178,71 @@ function flowview_display_report() {
 	swfobject.embedSWF('open-flash-chart.swf', 'chartpackets', '98%', height, '9.0.0', 'expressInstall.swf', {'data-file':'<?php print urlencode($config["url_path"] . "plugins/flowview/flowview.php?session=" . $sessionid . "&action=chartdata&exclude=" . get_request_var('exclude') . "&type=packets&title=$rname");?>', 'id':'chartpackets'});
 	swfobject.embedSWF('open-flash-chart.swf', 'chartflows', '98%', height, '9.0.0', 'expressInstall.swf', {'data-file':'<?php print urlencode($config["url_path"] . "plugins/flowview/flowview.php?session=" . $sessionid . "&action=chartdata&exclude=" . get_request_var('exclude') . "&type=flows&title=$rname");?>', 'id':'chartflows'});
 
-	$('#bytes').click(function() {
-		if (!$('#bytes').is(':checked')) {
-			$('#wrapperbytes').hide();
-			$.get('flowview.php?action=updatesess&type=bytes&value=');
+	$(function() {
```

Comment:
This is the actual change. Waiting for document ready.

Unfortunately, the diff looks like a lot changed, but the rest of the changes are spacing.